src/chordChart.ts

```diff
@@ -213,13 +213,19 @@ module powerbi.extensibility.visual {
             let columns: ChordChartColumns<ChordChartCategoricalColumns> = ChordChartColumns.getCategoricalColumns(dataView);
             let sources: ChordChartColumns<DataViewMetadataColumn> = ChordChartColumns.getColumnSources(dataView);
             let categoricalValues: ChordChartColumns<any> = ChordChartColumns.getCategoricalValues(dataView);
-
-            if (!categoricalValues || _.isEmpty(categoricalValues.Category) || _.isEmpty(categoricalValues.Y)) {
+            if (!categoricalValues || _.isEmpty(categoricalValues.Category)) {
                 return null;
             }
 
-            categoricalValues.Series = categoricalValues.Series || ChordChartColumns.getSeriesValues(dataView);
+            if (!categoricalValues.Y) {
+                categoricalValues.Y = [];
+                for (let i: number = 0; i < categoricalValues.Category.length; i++) {
+                    categoricalValues.Series.push(sources.Series.displayName + i);
+                    categoricalValues.Y.push(1);
```

Comment:
I don't really like this solution. Is it possible to update converter method as was done for Sankey?